GitHub Actions "PR Validation" is running DistributedLog tests

It looks like "PR Validation" is running tests, I was expecting that job to only run stuff like checkstyle, spotbugs....

I discovered this fact in #2228 because I saw a failed test

```
[ERROR] org.apache.distributedlog.common.config.TestConfigurationSubscription.testExceptionInConfigLoad(org.apache.distributedlog.common.config.TestConfigurationSubscription)
[ERROR]   Run 1: TestConfigurationSubscription.testExceptionInConfigLoad:169 » FileNotFound /tm...
[ERROR]   Run 2: TestConfigurationSubscription.testExceptionInConfigLoad:134 » IO Too many open...
[ERROR]   Run 3: TestConfigurationSubscription.testExceptionInConfigLoad:134 » IO Too many open...
[ERROR] org.apache.distributedlog.common.config.TestConfigurationSubscription.testReloadConfiguration(org.apache.distributedlog.common.config.TestConfigurationSubscription)
[ERROR]   Run 1: TestConfigurationSubscription.testReloadConfiguration:60 » IO Too many open fi...
[ERROR]   Run 2: TestConfigurationSubscription.testReloadConfiguration:60 » IO Too many open fi...
[ERROR]   Run 3: TestConfigurationSubscription.testReloadConfiguration:60 » IO Too many open fi...
[ERROR] Tests run: 10, Failures: 0, Errors: 2, Skipped: 0
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.21.0:test (default-test) on project distributedlog-common: There are test failures.
[ERROR] 
[ERROR] Please refer to /home/runner/work/bookkeeper/bookkeeper/stream/distributedlog/common/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :distributedlog-common
##[error]Process completed with exit code 1.
```

--- a/circe-checksum/pom.xml
+++ b/circe-checksum/pom.xml
@@ -86,6 +86,7 @@
               <include>**/*.cpp</include>
             </includes>
           </cpp>
+          <skipTests>${skipTests}</skipTests>
           <!-- skip by default, enabled in OS profiles -->
           <skip>true</skip>
         </configuration>

--- a/cpu-affinity/pom.xml
+++ b/cpu-affinity/pom.xml
@@ -53,6 +53,9 @@
         <groupId>com.github.maven-nar</groupId>
         <artifactId>nar-maven-plugin</artifactId>
         <extensions>true</extensions>
+        <configuration>
+          <skipTests>${skipTests}</skipTests>
+        </configuration>
       </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>

